changelog/updates/2022-04-07-google-gve.md

```diff
@@ -0,0 +1 @@
+- Enable GVE kernel option, which adds support for Google Virtual NIC on GCP (TODO: link)
```

Comment:
```suggestion
- Enabled GVE kernel driver, which adds support for Google Virtual NIC on GCP (TODO: link)
```
Maybe this file should be under `changes`, or?

Reply:
Reworded and moved to changes.